xml/System.ServiceModel.Syndication/SyndicationTextInput.xml

```diff
@@ -16,8 +16,36 @@
   </Base>
   <Interfaces />
   <Docs>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <summary>Represents a "textInput" entry from an RSS syndication feed.</summary>
+    <remarks>
+      <format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+ The RSS specification allows for a 'textInput' item to be included within a syndication feed that the publisher can use
+ to direct search requests or obtain feedback through a simple text input box and button. A 'textInput' element in an RSS feed
+ must contain 4 sub-elements:
+- title
+- description
+- name
+- link
+
+ If any of the four sub-elements are missing, 'textInput' will be null.
```

Comment:
If this is the `null` reserved keyword, then it should be:
```suggestion
 If any of the four sub-elements are missing, 'textInput' will be `null`.
```

Reply:
Also, if `textInput` happens to be the name of a metadata field (not a parameter, not a type), then it should also be wrapped like this:

```suggestion
 If any of the four sub-elements are missing, `textInput` will be `null`.
```

Reply:
Same suggestions would apply to the other similar remarks with a metadata and a null.